### PR TITLE
Add tab linking to URL

### PR DESCRIPTION
**Pull request by y-kkamil**
**Description**

Changes proposed in this pull request:

- Add ability to 'deep link' to a page with particular tab opened.
(there's an edge case in lambda details view - when you link to the 'testing' tab, and the lambda does not have an http trigger enabled, it will still display the tab. the solution would require more work, so i though i would leave it as it is, having in mind upcoming rework of this view)

**Related issue(s)**
#121

### CODE REVIEW

File: catalog/package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "private": true,
   "dependencies": {
     "@kyma-project/generic-documentation": "https://github.com/magicmatatjahu/console/raw/integrate-docs-component-2/components/generic-documentation/kyma-project-generic-documentation-0.1.0.tgz",
-    "@kyma-project/luigi-client": "0.4.9",
+    "@kyma-project/luigi-client": "0.6.0",
```
Comment (dariadomagala) [inline catalog/package.json]: Why only package.json? Shouldn't we add package-lock as well here? ;)

Comment (y-kkamil) [reply catalog/package.json]: you are totally right! :) i've added changed package-lock file

**Comment by y-kkamil** [comment]
> In Instances and Catalog going to url with preselected tab shows proper tab, but content is always the same, no matter which tabs was selected.

this is now fixed

File: service-catalog-ui/catalog/src/components/ServiceClassList/ServiceClassList.component.js [hunk 2]
```diff
@@ -56,7 +57,14 @@ class ServiceClassList extends React.Component {
     };
   }
 
-  setTabFilter = filterValue => {
+  setTabFilter = currentTabIndex => {
+    let filterValue;
```
[the remark this answers is not in the document]
Comment (y-kkamil) [reply service-catalog-ui/catalog/src/components/ServiceClassList/ServiceClassList.component.js]: i'd go for even simpler solution: 
```
  setTabFilter = currentTabIndex => {
    this.props.setServiceClassesFilter('local', currentTabIndex === 0);
  };
```

**Pull request merged**